rust/operator-binary/src/controller.rs

```diff
@@ -533,6 +542,7 @@ fn build_server_rolegroup_daemonset(
     rolegroup_ref: &RoleGroupRef<OpaCluster>,
     server_config: &HashMap<PropertyNameKind, BTreeMap<String, String>>,
     merged_config: &OpaConfig,
+    user_info_fetcher_image: &str,
```

Comment:
This is currently always the operator image, not sure if we want to keep it that way or split into a sidecar image (would be kinda painful with the dockerfile workflow...)

Reply:
if its painful lets leave it as is for now

Reply:
I agreed as well :D We worked hard to have product images that container everything to make product image selection easier